### ISSUE TITLE
NHS project

### ISSUE THREAD

**Issue by jsms90** [InFact-coop/InFact#22]
@nogainbar Please add any/all info here, so all 6 of us know what stage we're at :+1: 

Just some ideas:
+ time & date of calls / meetings
+ minutes from meetings
+ project brief
+ decisions/questions to throw back to the group
+ potential start dates

**Comment by jsms90** [comment]
You mentioned that you'd run some focus groups already?

**Comment by nogainbar** [comment]
Yes, I did two focus groups and attached is a PDF which I composed with the feedback. Do have a look at it!

[CAMHS:Rethink_Report.pdf](https://github.com/InFact-coop/InFact/files/941863/CAMHS.Rethink_Report.pdf)

- **time & date of calls / meetings:** two focus groups, one on the 29/10/16 and one on the 25/01/16
- Unfortunately I dont have minutes from meeting and I think @sofer mainly had phone contacts with them.
- **project brief:** TBC
- **decisions/questions to throw back to the group:** How to best project manage this project so that we end up doing it in Human Centered Design design process and build something really meaningful for them?
- **potential start dates**: TBC will try contacting them today and get back

**Comment by nogainbar** [comment]
@sofer will be meeting with them on Monday, I cannot unfortunately go because its the first day of UX/UI week and Dan suggested not to wait and that he will go.

I also asked @sofer to write down the ours he is putting in client management of this project as I would like him to be remunerated for it. 

We should expect to know more about this after the meeting, however please note that I will not have time to start working on the research and design phase of this project before **June**.

**Comment by jsms90** [comment]
@sofer said they're not looking to start til august?

**Comment by nogainbar** [comment]
@jsms90 and all, yes thats correct. the NHS project is going ahead but not until August. Workshop day however with PO and stakeholders beginning of june. We will just need to line up the projects in such a way that we have work before hand. meeting with Mark on Friday re the Anna Freud one. (#21)

**Comment by jsms90** [comment]
Notes from meeting
+ Combination of NHS & ReThink team
+ Project will need heavy amount of research & user testing (0.5 days per week for user testing?)
+ Budget: £20,000 for 5 sprints i.e. £4000 per sprint
  + June - one day workshop (**everyone** present)
  + August begin
    + 1 research/design sprint
    + 4 sprints
+ Roles
  + Project Management (@sohilpandya ?)
  + 1 week of UX research (@bradreeder @nogainbar ?)

**Action point** Client liason person needs to email client ASAP to agree date for June workshop & August start date.

**Comment by des-des** [comment]
Okay we have an action point that someone needs to follow up ASAP, but do we have a person? ..

**Comment by nogainbar** [comment]
Hey @des-des, that person is @sohilpandya :)

**Comment by des-des** [comment]
okay I have updated the comment

**Comment by sohilpandya** [comment]
I've emailed Imogen, will update as soon as I have a reply back from her.

**Comment by sohilpandya** [comment]
**update**

Had a call with Imogen and here is the update:
- Workshop to take place in the 3rd/4th week of June, going to finalize the date in the next week or so. 
- 3 ideas to be picked from the workshop 
- These will be narrowed down to 1 at a meeting involving FAC & Rethink
- Once the idea is picked @nogainbar and @bradreeder can go away and do their magic
- Ready for the development to start in the last week of July. 

How do these timelines sound?

**Comment by des-des** [comment]
@sohilpandya if @nogainbar is on holiday in July, should we try and make it in the 3rd week?

**Comment by nogainbar** [comment]
@des-des, yes as I understood we are starting in August and I was planning to do the UX research sprint on the first week of august. Im away from the 5-28 of july... I would say that development should start on the week starting on the 7th of august.

**Comment by nogainbar** [comment]
Workshop in June are OK, we must make sure however that @sofer is free to do those with us these dates.

**Comment by sohilpandya** [comment]
@des-des @nogainbar Thanks for the feedback, I have conveyed that to Imogen and she is happy with that format.
-One day workshop to take place in **June**, we come up with 3 ideas on the day that we like and then narrow down to the 1 which we want to develop. 
- First Sprint involving @nogainbar & @bradreeder to take place in first 2 weeks of August.
- Development to start in 3rd Week of August. 

The one-day workshop will take place between 20th - 30th June, she will get back to us with a confirmed date very soon. @sofer has to be available for it. ❤️ 


ps. this means we will have some developers in June, July and half of August with potentially no work. Lets get something sorted. get something in which is 3-4 sprints long. 😄

**Comment by bradreeder** [comment]
I'm going to be away from the 9th-20th August so I'll be able to cover the first week and a half. I hope this is okay @nogainbar .

**Comment by nogainbar** [comment]
@sohilpandya  thats brilliant, thank you.
@bradreeder thats not a problem. 

First week of August @bradreeder and I will spend some time in Rethink with the youngsters, clinicians and anyone involved to do interviews and user research.
I will then start the wireframing UX/UI between the first week and the end of the second week of august. 

The developers should have a flow and first phase design to work with on the week starting the **14th of August**.

@sohilpandya we also need to make sure we schedule testing groups and sessions with Imogen and the rethink people at the end of each sprint. 

Check infact coop calendar for updates.
@jsms90 @des-des @emilyb7

**Comment by des-des** [comment]
Okay, I have nothing planned june / july but would like to take holiday in august.. Can do a qa but keep it to a minimum

**Comment by sohilpandya** [comment]
@nogainbar Imogen has come back with 29th June as the date for the one day workshop. Can both you, @bradreeder and @sofer attend this? 

Please let me know so that I can get back to her asap. 

Thanks 👍

**Comment by bradreeder** [comment]
@sohilpandya That's the first week of FAC11, but it's a Thursday so should be fine as students will be working on projects. 👍

**Comment by sohilpandya** [comment]
Perfect! @sofer @nogainbar ?

**Comment by nogainbar** [comment]
yup! goog for me!!! @sohilpandya 🌟
